### PR TITLE
Fix docs to clarify the less than 256 limit for Padding().

### PR DESCRIPTION
**Pull request by Ayrx**
The current range implies that 256 is accepted.

### CODE REVIEW

File: docs/hazmat/primitives/padding.rst [hunk 1]
```diff
@@ -39,7 +39,7 @@ multiple of the block size.
     :param block_size: The size of the block in bits that the data is being
                        padded to.
     :raises ValueError: Raised if block size is not a multiple of 8 or is not
-        between 0 and 256.
+        between 0 and 255.
```
Comment (cedk) [inline docs/hazmat/primitives/padding.rst]: Is it not even less ambiguous to say: 255 included.

Comment (Ayrx) [inline docs/hazmat/primitives/padding.rst]: Good point.

**Comment by cedk** [comment]
LGTM

**Comment by alex** [comment]
Jenkins, retest this please.

**Comment by reaperhulk** [comment]
This is related to #3098

**Comment by Ayrx** [comment]
@reaperhulk This can be merged regardless of the outcome on #3098 since it applies to the PKCS7 padding either way. :)

**Comment by reaperhulk** [comment]
Agreed, and this LGTM

**Pull request merged**